libhb/stream.c

```diff
@@ -3961,8 +3961,8 @@ static void hb_ps_stream_find_streams(hb_stream_t *stream)
                 // (which do not have a program stream map)  may use
                 // this for other types of video.
                 //
-                // Also, the hddvd tards decided to use 0xe2 and 0xe3 for
-                // h.264 video :( and the twits decided not to put a
+                // Also, the hddvd folks decided to use 0xe2 and 0xe3 for
+                // h.264 video :( and the folks decided not to put a
```

Comment:
This project has a code of conduct file, and if anyone were to make a PR to add the lines in pink here, I'm fairly confident that PR would be in violation of that file.

I've picked a fairly benign word to replace both terms. You're of course free to do whatever you like to any or all of these changes, including selecting alternate replacements. You could even choose to call me by any of these pink highlighted words, however, I believe that act too would be in violation of that document.